calico/felix/config.py

```diff
@@ -163,6 +166,14 @@ def validate_cfg(self):
         self.validate_addr("PluginAddress", self.PLUGIN_ADDR)
         self.validate_addr("ACLAddress", self.ACL_ADDR)
 
+        #*********************************************************************#
+        #* Bind address must be * or an IP address.                          *#
+        #*********************************************************************#
+        if (self.BIND_ADDR != "*" and
+                not common.validate_ipv4_addr(self.BIND_ADDR)):
```

Comment:
I think it probably does make sense to specify a hostname, though it feels a bit odd - so I will change that.